### ISSUE TITLE
ModuleNotFoundError: No module named 'watchfiles._rust_notify'

### ISSUE THREAD

**Issue by lapinvert** [adamchainz/django-watchfiles#85]
### Python Version

3.10.12

### Django Version

4.2.4

### Package Version

0.1.1

### Description

I get the error `ModuleNotFoundError: No module named 'watchfiles._rust_notify'` right after starting `runserver`.

I am running it on an Mac OS M1 machine, and through a docker image that is based on Ubuntu (`FROM ubuntu:22.04`).

Any ideas how to solve it?

**Comment by adamchainz** [comment]
This sounds like a watchfiles issue, like it didn't properly install. Try reinstalling it, then check in its issue tracker?

Also in the future share a full stack trace not just the exception message.

**Comment by StevenMapes** [comment]
I think the problem they had is related to #12 as I've recreated their error by installing django-watchfiles and watchfiles then when the runserver is called I get the below output

```
[2024-09-09 09:21:20 -INFO] Watching for file changes with WatchfilesReloader
Traceback (most recent call last):
  File "~/projects/example/manage.py", line 21, in <module>
    main()
  File "~/projects/example/manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django/core/management/base.py", line 413, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django/core/management/commands/runserver.py", line 75, in execute
    super().execute(*args, **options)
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django/core/management/base.py", line 459, in execute
    output = self.handle(*args, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django/core/management/commands/runserver.py", line 112, in handle
    self.run(**options)
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django/core/management/commands/runserver.py", line 119, in run
    autoreload.run_with_reloader(self.inner_run, **options)
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django/utils/autoreload.py", line 671, in run_with_reloader
    start_django(reloader, main_func, *args, **kwargs)
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django/utils/autoreload.py", line 660, in start_django
    reloader.run(django_main_thread)
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django/utils/autoreload.py", line 344, in run
    self.run_loop()
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django/utils/autoreload.py", line 350, in run_loop
    next(ticker)
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django_watchfiles/__init__.py", line 89, in tick
    for changes in self.watcher:
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/django_watchfiles/__init__.py", line 40, in __iter__
    for changes in watchfiles.watch(
  File "/home/user/.pyenv/versions/example-venv/lib/python3.11/site-packages/watchfiles/main.py", line 119, in watch
    with RustNotify(
         ^^^^^^^^^^^
FileNotFoundError: No path was found.
```

Having read into issue #12 it's appears to be  caused by this issue as if I then apply the changes by [wsantos](https://github.com/adamchainz/django-watchfiles/issues/12#issuecomment-2310616815) the error no longer persists so I'm 90% sure this is the same cause

**Comment by adamchainz** [comment]
Thank you @StevenMapes . I will close this since there was no follow up.

**Comment by lapinvert** [comment]
@adamchainz Sorry, I think I just removed the tool, moved on, and forgot to reply here. I tried again with a new project and a fresh dev environment, and I can still reproduce the error, but I'm doing something that doesn’t really make sense (running `runserver` on host machine, when `.venv` was built on docker container (Ubuntu based). I can't remember if I was getting that mixed up last time when I reported the bug.

So this time, I tried again, running runserver from my dev container like usual, and what happens is `Watching for file changes with WatchfilesReloader` just doesn’t show up. No errors, nothing, just that line, which should confirm it’s working, is missing.

I checked and It does show up when running the same project on a local machine without a dev container, but not when using one.
